messaging-core/src/main/webapp/WEB-INF/messaging/newMessage.jsp

```diff
@@ -5,6 +5,9 @@
 <%@ taglib uri="org.fenixedu.messaging.tags.sorter" prefix="sort" %>
 ${portal.toolkit()}
 
+<link href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/css/select2.min.css" rel="stylesheet" />
```

Comment:
Should the select2 import be in the jsp or should it be available in the local directories?